src/app/core/account/account.service.ts

```diff
@@ -12,15 +12,27 @@ export interface Account {
 
 @Injectable()
 export class AccountService {
+  // tslint:disable-next-line: variable-name
+  _accounts: BehaviorSubject<Account[]> = new BehaviorSubject([]);
+
+  get accounts(): Observable<Account[]> {
+    return this._accounts.asObservable();
```

Comment:
I turned accounts into a BehaviorSubject so the accounts are loaded by the container component and other components can just subscribe to the list of accounts.

Reply:
+1